components/EditDeleteInfoForm/index.js

```diff
@@ -1,4 +1,4 @@
-import React, { useState, useEffect, useCallback } from "react";
+import React, { useState, useEffect } from "react";
```

Comment:
You don't seem to be using ```React``` in this component (you no longer need to import) so i think we can remove that. 

```suggestion
import { useState, useEffect } from "react";
```